File: Java/FilmBioskop.java
public class FilmBioskop extends Film {
    private String studio;
    private String jadwal;
    private int harga;

    public FilmBioskop(int id, String judul, int durasi, String genre, String sutradara, String klasifikasi,
                       String studio, String jadwal, int harga){
        super(id, judul, durasi, genre, sutradara, klasifikasi);
        this.studio = studio;
        this.jadwal = jadwal;
        this.harga = harga;
    }
    public String getStudio(){ return studio; }
    public String getJadwal(){ return jadwal; }
    public int getHarga(){ return harga; }
    public void setStudio(String studio){ this.studio = studio; }
    public void setJadwal(String jadwal){ this.jadwal = jadwal; }
    public void setHarga(int harga){ this.harga = harga; }
}

File: Python/filmbioskop.py
from film import Film

class FilmBioskop(Film):
    def __init__(self, id, judul, durasi, genre, sutradara, klasifikasi, studio, jadwal, harga):
        super().__init__(id, judul, durasi, genre, sutradara, klasifikasi)
        self.studio = studio
        self.jadwal = jadwal
        self.harga = harga

    def get_studio(self):
        return self.studio

    def get_jadwal(self):
        return self.jadwal

    def get_harga(self):
        return self.harga

    def set_studio(self, studio):
        self.studio = studio

    def set_jadwal(self, jadwal):
        self.jadwal = jadwal

    def set_harga(self, harga):
        self.harga = harga